packages/service-clients/end-to-end-tests/azure-client/src/test/containerCreate.spec.ts

```diff
@@ -189,3 +191,99 @@ describe("Container create with feature flags", () => {
 		assert(featureGates.length > 0);
 	});
 });
+
+/**
+ * Testing scenarios for creating/loading containers with the legacy (LTS) version of AzureClient.
+ */
+describe("Container create with legacy version", () => {
+	const connectTimeoutMs = 10_000;
+	let clientCurrent: AzureClient;
+	let clientLegacy: AzureClientLegacy;
+	const schemaCurrent = {
+		initialObjects: {
+			map1: SharedMap,
+		},
+	} satisfies ContainerSchema;
+
+	const schemaLegacy = {
+		initialObjects: {
+			map1: SharedMapLegacy,
+		},
+	};
+
+	beforeEach("createAzureClients", () => {
+		clientCurrent = createAzureClient();
+		clientLegacy = createAzureClientLegacy();
+	});
+
+	/**
+	 * Scenario: test if a legacy AzureClient can get a container made by the current AzureClient.
+	 *
+	 * Expected behavior: an error should not be thrown nor should a rejected promise
+	 * be returned.
+	 */
+	it("Legacy AzureClient can get container made by current AzureClient", async () => {
+		const { container: container1 } = await clientCurrent.createContainer(schemaCurrent);
```

Comment:
Maybe consider naming these `currentContainer` and `legacyContainer` for easier reading

```suggestion
		const { container: currentContainer } = await clientCurrent.createContainer(schemaCurrent);
```